website/build_site.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+ROOT_DIR=`pwd`
+
+WORK_DIR=/website/public
+ME=`basename $0`
+
+rm -rf $WORK_DIR
+mkdir -p $WORK_DIR
+
+cd website
+
+make setup
+
+make site
+
+# push all of the results to asf-site branch
```

Comment:
Maybe breaking the code below into a publish_website.sh could be more flexible.

Reply:
I agree